sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/DatabricksLiveContainerMigration/CosmosDBLiveContainerMigration.scala

```diff
@@ -0,0 +1,81 @@
+// Databricks notebook source
+  val cosmosEndpoint_cf = "" //enter your Cosmos DB Account URI
+  val cosmosMasterKey_cf = "" //enter your Cosmos DB Account PRIMARY KEY
+  val cosmosDatabaseName_cf = "database-v4" //replace database-v4 with the name of your source database
+  val cosmosContainerName_cf = "customer" //replace customer with the name of the container you want to migrate
+  val cosmosContainerName_throughputControl = "ThroughputControl"
+
+  val cosmosEndpoint_write = "" //enter your Cosmos DB Account URI
+  val cosmosMasterKey_write = "" //enter your Cosmos DB Account PRIMARY KEY
+  val cosmosDatabaseName_write = "database-v4" //replace this with the name of your target database
+  val cosmosContainerName_write = "customer_v2" //replace this with what you want to name your target container
+
+// COMMAND ----------
+
+spark.conf.set("spark.sql.catalog.cosmosCatalog", "com.azure.cosmos.spark.CosmosCatalog")
+spark.conf.set("spark.sql.catalog.cosmosCatalog.spark.cosmos.accountEndpoint", cosmosEndpoint_cf)
+spark.conf.set("spark.sql.catalog.cosmosCatalog.spark.cosmos.accountKey", cosmosMasterKey_cf)
+
+// COMMAND ----------
+
+// MAGIC %sql
+// MAGIC /* NOTE: It is important to enable TTL (can be off/-1 by default) on the throughput control container */
+// MAGIC CREATE TABLE IF NOT EXISTS cosmosCatalog.`database-v4`.ThroughputControl -- replace database-v4 with source database name - ThroughputControl table will be created there
+// MAGIC USING cosmos.oltp
+// MAGIC OPTIONS(spark.cosmos.database = 'database-v4') -- replace database-v4 with the name of your source database
+// MAGIC TBLPROPERTIES(partitionKeyPath = '/groupId', autoScaleMaxThroughput = '4000', indexingPolicy = 'AllProperties', defaultTtlInSeconds = '-1');
+// MAGIC 
+// MAGIC CREATE TABLE IF NOT EXISTS cosmosCatalog.`database-v4`.customer_v2 -- replace database-v4 with the name of your source database, and customer_v2 with what you want to name your target container - it will be created here
+// MAGIC USING cosmos.oltp 
+// MAGIC -- replace /customerId with the name of the field that you want to be used as the partition key in the new version of the container
+// MAGIC TBLPROPERTIES(partitionKeyPath = '/customerId', autoScaleMaxThroughput = '100000', indexingPolicy = 'OnlySystemProperties');
+
+// COMMAND ----------
+
+  val changeFeedCfg = Map("spark.cosmos.accountEndpoint" -> cosmosEndpoint_cf,
+    "spark.cosmos.accountKey" -> cosmosMasterKey_cf,
```

Comment:
Please add entry with `spark.cosmos.applicationName` here and in the write cfg below
It will be used as USerAgentSuffix - so we can identify how many customers have used this sample

So something like LiveMigrationRead_<someGuid> and LivemigrationWrite_<SameGuid>

Reply:
Ok